request_handlers.h

```diff
@@ -0,0 +1,13 @@
+#ifndef REQUEST_HANDLER_H
+#define REQUEST_HANDLER_H
+
+#include <boost/asio.hpp>
+namespace bai = boost::asio::ip;
```

Comment:
Would it be clearer to declare `using boost::asio::ip;` instead of creating a custom `bai` name? For example, we can glance up to see the definition of `bai` here, but its meaning is non-obvious in any file that includes this header.